debezium-core/src/main/java/io/debezium/config/Configuration.java

```diff
@@ -2125,6 +2125,15 @@ default String getFallbackStringProperty(Field newProperty, Field oldProperty) {
      * @return the evaluated value
      */
     static String getFallbackStringProperty(Configuration config, Field newProperty, Field oldProperty) {
```

Comment:
This method seems only to be used by the instance method right above. So let's inline this one, leaving only the instance method behind.

Reply:
I think this static method is used on many other places (that's why we also need a logging and non-logging version of the code. For example see Filters, RelationalTableFilters, *ConnectoConfig implementations, etc, isn't it?